apps/meteor/ee/app/license/server/license.ts

```diff
@@ -223,7 +223,7 @@ class LicenseClass {
 		EnterpriseLicenses.emit('invalidate');
 	}
 
-	async canAddNewUser(userCount = 1): Promise<boolean> {
+	async canAddNewUser(userCount = 0): Promise<boolean> {
```

Comment:
This function checks if adding a new user would cause the limit to be crossed, so 1 **is** the right value here. If this is causing issues, then something must be using this function in the wrong way.

Reply:
The error is inside the function; It should be doing `maxActiveUsers >=`, not `maxActiveUsers >`